airflow/models/dag.py

```diff
@@ -2698,10 +2698,16 @@ def add_logger_if_needed(ti: TaskInstance):
         # than creating a BackfillJob and allows us to surface logs to the user
         while dr.state == State.RUNNING:
             schedulable_tis, _ = dr.update_state(session=session)
-            for ti in schedulable_tis:
-                add_logger_if_needed(ti)
-                ti.task = tasks[ti.task_id]
-                _run_task(ti, session=session)
+            try:
+                for ti in schedulable_tis:
+                    add_logger_if_needed(ti)
+                    ti.task = tasks[ti.task_id]
+                    _run_task(ti, session=session)
+            except Exception:
+                self.log.info(
+                    "Task failed. DAG will continue to run until finished "
+                    "and be marked as failed."
+                )
```

Comment:
```suggestion
                self.log.info(
                    "Task failed. DAG will continue to run until finished "
                    "and be marked as failed.",
                    exc_info=True,
                )
```

If we want to include the exception.